rudder-agent/SOURCES/patches/cfengine/02-sighup-support-in-cfserverd.patch

```diff
@@ -1,82 +0,0 @@
---- a/cf-serverd/cf-serverd-functions.c	2014-11-26 06:26:51.000000000 +0100
-+++ b/cf-serverd/cf-serverd-functions.c	2014-11-28 15:10:21.624969933 +0100
-@@ -356,6 +356,14 @@
-     Log(LOG_LEVEL_DEBUG, "Checking file updates for input file '%s'",
-         config->input_file);
- 
-+    /* If we force the reload of config, effectively recheck the promises */
-+    if ( IsRequestReloadConfig() )
-+    {
-+        Log(LOG_LEVEL_VERBOSE, "Force reload of inputs files...");
-+        GenericAgentCheckPolicy(config, true, true);
```

Comment:
Unsure about this one. Config reload on SIGHUP was added but it is done differently, and this [comment](https://github.com/cfengine/core/blob/3.7.0/cf-serverd/cf-serverd-functions.c#L400) is not reassuring.